test/SlowTests/Issues/RavenDB_19548.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Linq;
+using Raven.Client.Documents.Queries;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_19548 : RavenTestBase
+    {
+        public RavenDB_19548(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task TestCase()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var session = store.OpenAsyncSession())
+                {
+                    await session.StoreAsync(new TestObj { Name = "Omer", Prop1 = "123456", Prop2 = "12", Birthday = new DateTime(1994, 3, 22) });
+                    await session.SaveChangesAsync();
+                }
+
+                using (var session = store.OpenAsyncSession())
+                {
+                    var asyncDocumentQuery = from result in session.Query<TestObj>() select result.Prop1.Length;
+                    Console.WriteLine(asyncDocumentQuery);
```

Comment:
Remove this

Reply:
Add an assert on the query text, so it is easier to see what it is doing here.
(And in all the other locations as well)

Reply:
Done